extensions/wikia/EditPageLayout/templates/EditPageLayout_EditPage.php

```diff
@@ -87,23 +87,40 @@
 					</noscript>
 					<div class="module module_page_controls">
 						<div class="module_content">
-							<div class="checkboxes">
-								<?php foreach( $customCheckboxes as $i=>$checkbox ): ?>
-									<label class="<?= $checkbox['name'] ?>">
-										<input type="checkbox" name="<?= $checkbox['name'] ?>" tabindex="<?= $i + 2 ?>" id="<?= $checkbox['name'] ?>" <?= $checkbox['checked'] ? ' checked="checked"' : '' ?> />
-										<?= $checkbox['label'] ?>
-									</label>
-								<?php endforeach ?>
-								<?php if ( $canMinorEdit ): ?>
-									<label class="wpMinoredit">
-										<input type="checkbox" tabindex="21" name="wpMinoredit" id="wpMinoredit" accesskey="<?=wfMsg('accesskey-minoredit');?>"<?= $minorEditCheckbox ? ' checked="checked"' : '' ?> />
-										<?= wfMsg('editpagelayout-pageControls-minorEdit') ?>
-									</label>
-								<?php endif ?>
+							<div class="wpSummaryFields">
+								<div class="checkboxes">
+									<?php foreach( $customCheckboxes as $i=>$checkbox ): ?>
+										<label class="<?= $checkbox['name'] ?>">
+											<input type="checkbox" name="<?= $checkbox['name'] ?>" tabindex="<?= $i + 2 ?>" id="<?= $checkbox['name'] ?>" <?= $checkbox['checked'] ? ' checked="checked"' : '' ?> />
+											<?= $checkbox['label'] ?>
```

Comment:
Are the items in `$checkbox` always safe to output (i.e. are they already escaped)?